src/__experimental__/components/radio-button/radio-button-group.spec.js

```diff
@@ -8,17 +8,30 @@ import { StyledRadioButtonGroup } from './radio-button.style';
 import Label from '../label';
 import LabelStyle from '../label/label.style';
 import { assertStyleMatch } from '../../../__spec_helper__/test-utils';
+import ValidationIconStyle from '../../../components/validations/validation-icon.style';
+import baseTheme from '../../../style/themes/base';
+import FormFieldStyle from '../form-field/form-field.style';
 
 const buttonValues = ['test-1', 'test-2'];
 const groupName = 'test-group';
 
 function render(renderer = TestRenderer.create) {
-  const children = buttonValues.map(value => <RadioButton value={ value } />);
+  const children = buttonValues.map((value, index) => (
+    <RadioButton
+      id={ `rId-${index}` }
+      key={ `radio-key-${value}` }
```

Comment:
since we have prefix on key, maybe its better to go with shorter `index` instead of `value`?

Reply:
@damianOfKyan Eslint throws a warning (react/no-array-index-key) on 
```
key={ `radio-key-${index}` }
```
and
```
key={ index }
```

What I've done gets around that warning.